### PR TITLE
Create greetings.yaml

### PR DESCRIPTION
**Pull request by sanmarg**
Added for #45

### CODE REVIEW

File: .github/workflows/greetings.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (Witty-Wizard) [inline .github/workflows/greetings.yaml:14]: change the token to `GH-GREETING`

**Review by Witty-Wizard — CHANGES_REQUESTED** [review]

File: .github/workflows/greetings.yaml (hunk 1 above)
Comment (sanmarg) [reply .github/workflows/greetings.yaml:14]: Sure ill do it

**Comment by sanmarg** [comment]
Done with the update of the token.

**Review by Witty-Wizard — APPROVED** [review]
Great

**Pull request merged**